db/src/main/resources/db/migration/V1_020__add_address_column.sql

```diff
@@ -0,0 +1,3 @@
+alter table claimant add column address_id uuid;
```

Comment:
Why not just use rename column? Then it's a one liner

Reply:
Because during blue/green deployment the old version of the app needs to be able to write to the old column.